Tests/SwiftParserTest/translated/GenericDisambiguationTests.swift

```diff
@@ -237,8 +237,7 @@ final class GenericDisambiguationTests: XCTestCase {
   func testGenericDisambiguation15() {
     AssertParse(
       """
-      // TODO: parse empty <> list
-      //A<>.c() // e/xpected-error{{xxx}}
+      A<>.c()
```

Comment:
What tree does this produce?

Reply:
Thanks for checking. It parses `<>` as an operator. I’ll leave this TODO in for now.

Reply:
Would be good to keep a TODO here (or did you open a bug?)

Reply:
I wonder where I lost that change to keep the TODO. It’s in again now.